torch_frame/data/multi_nested_tensor.py

```diff
@@ -275,6 +282,27 @@ def _single_index_select(self, index: int, dim: int) -> MultiNestedTensor:
         else:
             raise RuntimeError(f"Unsupported dim={dim} for index_select.")
 
+    def fillna_col(
+        self,
+        index: int,
+        fill_value: Union[int, float, Tensor],
+    ) -> None:
```

Comment:
Add docs that we inplace modify the tensor?

Reply:
+1. I don't get the behavior even from the test.